harper-core/build.rs

```diff
@@ -0,0 +1,47 @@
+use std::fs::File;
+use std::io;
+
+use fst::MapBuilder;
+use harper_dictionary_parsing::{
+    parse_default_attribute_list, parse_default_word_list, CharString,
+};
+use hashbrown::HashMap;
+use itertools::Itertools;
+
+fn main() {
+    let wtr = io::BufWriter::new(File::create("dictionary.fst").unwrap());
+    let mut build = MapBuilder::new(wtr).unwrap();
```

Comment:
If we move the dictionaries into `harper-dictionary-parsing`, could we use `FstDictionary::new()` instead of building it manually?

Reply:
can you elaborate on what you mean by this?

Reply:
Nevermind, I think I get what you're saying. Yeah, I think we can do that.